src/thi/ng/geom/polygon.cljc

```diff
@@ -364,7 +365,19 @@
     [{points :points} t] (gu/point-at t (conj points (first points))))
   (random-point
     [_] (g/point-at _ (m/random)))
-  (random-point-inside [_] nil) ; TODO
+  ;; Uniformly sample point from tessellated triangles of polygon
+  ;; https://blogs.sas.com/content/iml/2020/10/21/random-points-in-polygon.html
+  ;; https://observablehq.com/@scarysize/finding-random-points-in-a-polygon
+  (random-point-inside
+    [_]
+    "Uniformly sample point from inside polygon
+
+    Tessellates into triangles, randomly selects a triangle weighted by area,
+    and then samples a point inside of that triangle uniformly."
+    (->> (g/tessellate _)
+         (map t/triangle2)
```

Comment:
If https://github.com/thi-ng/geom/pull/89 is merged, this conversion to triangles will no longer be necessary.